extension/content.js

```diff
@@ -51,6 +52,24 @@ function addReleasesTab() {
 	}
 }
 
+function addBlameParentLinks() {
+	let pathname = location.pathname;
+
+	$('.blame-sha').each(function(index, commitLink) {
+		let $commitLink = $(commitLink)
+		let $blameParentLink = $commitLink.clone();
+		let commitSha = commitLink.href.match(/(?<=commit\/)\w+/)[0];
```

Comment:
@hkdobrev to improve that a little you probably want something more like this:

```js
var re = /(\/(commit)\/\w{40}$)/
```

this is `/commit/[40 char sha][eol]`